spec/domain/operations/people/create_or_update_spec.rb

```diff
@@ -36,6 +36,16 @@
           expect(@result).to be_a(Dry::Monads::Result::Success)
         end
       end
+
+      context 'nil hbx id' do
+        before :each do
+          person_params.merge!({person_hbx_id: nil})
+          @result = subject.call(params: person_params)
+        end
+        it 'should return success' do
+          expect(@result).to be_a(Dry::Monads::Result::Success)
```

Comment:
Tested this locally and the spec still passes with the original code. Let's find a way to make this fail with the original code and pass with the new changes.